src/components/download-overlay/download-overlay.tsx

```diff
@@ -103,39 +118,42 @@ const DownloadOverlay = withText({
         const shouldRenderSources = downloadConfig.displaySources && (downloadMetadata!.flavors.length || downloadMetadata!.imageDownloadUrl);
         const shouldRenderCaptions = downloadConfig.displayCaptions && downloadMetadata!.captions.length;
         const shouldRenderAttachments = downloadConfig.displayAttachments && downloadMetadata!.attachments.length;
+        const isAudioPreset = activePreset === ReservedPresetNames.MiniAudioUI;
```

Comment:
i dont think that download plugin should know about audio plugin
what if tomorrow we want to use download plugin inside another plugin ?

Reply:
removed